schema/tests/Provider/Company/CompanyLifeCycleTest.php

```diff
@@ -282,6 +284,80 @@ public function removing_company_removes_domain()
         $this->assertNull($company->getDomain());
     }
 
+    public function added_company_has_default_mediaRelaySetId()
+    {
+        $companyDto = new CompanyDto();
+        $companyDto
+            ->setName('ACompany')
+            ->setDomainUsers('127.3.0.1')
+            ->setInvoicingNif('12345678B')
+            ->setMaxCalls(0)
+            ->setInvoicingPostalAddress('An address')
+            ->setInvoicingPostalCode('54321')
+            ->setInvoicingTown('')
+            ->setInvoicingProvince('')
+            ->setInvoicingCountryName('')
+            ->setIpfilter(false)
+            ->setOnDemandRecord(0)
+            ->setOnDemandRecordCode('')
+            ->setExternallyextraopts('')
+            ->setRecordingsLimitEmail('')
+            ->setLanguageId(1)
+            ->setMediaRelaySetsId(null)
+            ->setDefaultTimezoneId(1)
+            ->setBrandId(1)
+            ->setDomainId(1)
+            ->setCountryId(1)
+            ->setApplicationServerSetId(1)
+            ->setMediaRelaySetId(null);
+
+        /** @var Company $as */
+        $company = $this->entityTools
+            ->persistDto($companyDto, null, true);
+
+        $this->assertEquals(
+            0,
+            $company->getMediaRelaySet()->getId()
+        );
+    }
+
+    public function added_company_has_default_applicationServerSetId()
+    {
+        $companyDto = new CompanyDto();
+        $companyDto
+            ->setName('ACompany')
+            ->setDomainUsers('127.3.0.1')
+            ->setInvoicingNif('12345678B')
+            ->setMaxCalls(0)
+            ->setInvoicingPostalAddress('An address')
+            ->setInvoicingPostalCode('54321')
+            ->setInvoicingTown('')
+            ->setInvoicingProvince('')
+            ->setInvoicingCountryName('')
+            ->setIpfilter(false)
+            ->setOnDemandRecord(0)
+            ->setOnDemandRecordCode('')
+            ->setExternallyextraopts('')
+            ->setRecordingsLimitEmail('')
+            ->setLanguageId(1)
+            ->setMediaRelaySetsId(null)
+            ->setDefaultTimezoneId(1)
+            ->setBrandId(1)
+            ->setDomainId(1)
+            ->setCountryId(1)
+            ->setApplicationServerSetId(null)
+            ->setMediaRelaySetId(1);
+
+        /** @var Company $as */
```

Comment:
remove please

Reply:
I change it to
` /** @var Company $company */`

Every reference to other entities in this orm tests it's done by concrete classes, not interfaces.